src/main/java/Controllers/GenericServlet.java

```diff
@@ -12,13 +12,17 @@
 
 public class GenericServlet extends HttpServlet {
     protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
-
     }
 
+    @Override
     protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
         Connection con = Database.getInstance().getConnection();
+        System.out.println("I am hereree");
```

Comment:
These lines are nice for debugging, but normally I would suggest that they are removed before creating a pull request as the strings have no useful information and may not be used by other developers.

In Android we use *Console.log* for things like this if they are absolutely necessary. I'm sure there's something similar in standard Java.

I'll approve the PR but I'll likely try to remove these in a future PR.